<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchLPDialog</class>
 <widget class="QDialog" name="BatchLPDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add multiple labeling positions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>name template:</string>
       </property>
       <property name="buddy">
        <cstring>nameTemplateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameTemplateEdit">
       <property name="text">
        <string>{chainID}{resID}</string>
       </property>
       <property name="placeholderText">
        <string>{resid}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Copy parameters from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="settingSource"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>strip mask:</string>
       </property>
       <property name="buddy">
        <cstring>nameTemplateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stripTemplateEdit">
       <property name="text">
        <string>chain {chainID} and resid {resID} and not name CA CB C N O</string>
       </property>
       <property name="placeholderText">
        <string>{resid}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Residues to label:</string>
       </property>
       <property name="buddy">
        <cstring>residuesWidget</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="allResidues">
       <property name="text">
        <string>all residues</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="residuesWidget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchLPDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchLPDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allResidues</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BatchLPDialog</receiver>
   <slot>toggleAll(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleAll(int)</slot>
 </slots>
</ui>
